docs/UserGuide.md

```diff
@@ -17,12 +16,60 @@ title: User Guide
 ### 1.1 What is HobbyList?
 HobbyList is a **desktop app for managing hobby activities, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, HobbyList can get your activity management tasks done faster than traditional GUI apps.
 
+If you prefer typing only and want to record your experiences with your hobby activities such as reading books or watching movies, HobbyList is the perfect app for you.
+
 --------------------------------------------------------------------------------------------------------------------
 
 ## 2. About this User Guide
 
-### 2.1 How to use this user guide?
-to be added
+### 2.1 What is this user guide for?
+This user guide provides you with all the information required for you to become adept at using HobbyList. You will learn all the features that HobbyList has to offer and how to make use of them to the fullest to keep records of all your hobby activities. There are information on the basic commands such as [add](#51-adding-a-hobby-activity--add), [delete](#53-deleting-an-activity--delete), [list](#54-listing-all-hobby-activities--list) and [rate](#55-rating-an-activity-rate), and also advanced features such as [renaming command names](#514-renaming-the-commands).
+
+### 2.2 How to use this user guide?
+
+#### Table of contents
+The [table of contents](#toc) is at the top of the page. From there, you can click on any of the sections to jump to it.
+
+#### Quick start
+If you want to **quick**ly get **start**ed with using HobbyList, you can read and follow the instructions in the [Quick start](#3-quick-start) section
+
+#### Features
+The [features](#5-features) section contains a list of all the features that HobbyList currently has. Within each feature, you can read up on how each feature is used and look at some examples.
+The features are laid out with a short description, followed by the format of the command, then finally some examples with explanation on what they do.
+This [information box](#command-info) provides information on understanding the command format. This [information box](#what-is-an-activity) tells you what an activity is and what information is stored in one.
```

Comment:
These two `information box` is quite confusing for new users. It would be nice if you can rename these.